channels/it.m3u

```diff
@@ -470,7 +470,7 @@ http://193.70.81.40:1935/supertvoristano/_definst_/supertvoristano/playlist.m3u8
 https://viacomitalytest-lh.akamaihd.net/i/sbshdlive_1@357018/master.m3u8
 #EXTINF:-1 tvg-id="SuperSixLombardia.it" tvg-name="SuperSix Lombardia" tvg-country="IT" tvg-language="" tvg-logo="" group-title="",SuperSix Lombardia (720p)
 https://5db313b643fd8.streamlock.net/SUPERSIXLombardia/SUPERSIXLombardia/playlist.m3u8
-#EXTINF:-1 tvg-id="SupertoonsTV.it" tvg-name="Supertoons TV" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/35BpWJh.jpg" group-title="",Supertoons TV (720p)
+#EXTINF:-1 tvg-id="SupertoonsTV.it" tvg-name="Supertoons TV" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/35BpWJh.jpg" group-title="Animation",Supertoons TV (720p)
```

Comment:
Same here

by the way... why are these Supertoons divided into all the different countries whilst the others aren't.

Reply:
Seems to be based out of the UK.

https://www.kedoo.com/en/kids/

Reply:
Should I change the county to UK but leave the language set?